src/main/java/swervelib/SwerveDrive.java

```diff
@@ -924,7 +941,12 @@ public void updateOdometry() {
     odometryLock.lock();
     try {
       // Update odometry
-      swerveDrivePoseEstimator.update(getYaw(), getModulePositions());
+        final Rotation2d yaw = getYaw();
+      final SwerveModulePosition[] modulePositions = getModulePositions();
+      swerveDrivePoseEstimator.update(yaw, modulePositions);
+      _swerveDrivePoseEstimator_NoVision_.update(yaw, modulePositions);
+      reportVisionVsNoVisionDelta(swerveDrivePoseEstimator.getEstimatedPosition(),
+          _swerveDrivePoseEstimator_NoVision_.getEstimatedPosition());
```

Comment:
The "no vision" pose estimator is updated with the wheel odometry but not with the vision measurements.
We immediately report the delta between the two poses (with and without vision).